app/src/main/java/com/example/teamyuml/decentworkmobile/fragments/NoticeForm.java

```diff
@@ -49,20 +56,61 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        Bundle initClass = getArguments();
```

Comment:
1. Why you initialize it here? I mean you start using it like 20 lines later.
2. `initClass` means that you want initialize the class and I dont't think you want to do that. You rather want to map some arguments from one activity/fragment to this one. So you rather names like: `fields`, `arguments`, `bundle` rather than `initClass` IMO